### PR TITLE
Added positronium notes

### PR DESCRIPTION
**Pull request by andrewfullard**
### :pencil: Description

**Type:** :memo: `documentation`

Added some documentation for positronium discussion. Probably won't stay under the "energy input" header, but it's the best place to keep a record.

### :pushpin: Resources

See https://en.wikipedia.org/wiki/Positronium 
Positronium spectrum https://ui.adsabs.harvard.edu/abs/1949PhRv...75.1696O 
SN Ia commentary (section 2.2.2) https://ui.adsabs.harvard.edu/abs/2004ApJ...613.1101M/abstract

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [x] I requested two reviewers for this pull request
- [x] I updated the documentation according to my changes
- [x] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: docs/physics/energy_input/positronium.rst
Comment (smithis7) [inline docs/physics/energy_input/positronium.rst:1]: line of *** is too long, should be ``***********`` on top and bottom (one * for each character in the title, otherwise sphinx complains).

**Review by isaacgsmith — CHANGES_REQUESTED** [review]
Can we hold off on this until #2105 is merged? That will give us a good location for both the gammaray deposition and the positronium docs.

**Pull request merged**
